ecommerce/frontend/src/components/ShoppingCart/ShoppingCartMaster.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component, useState } from 'react'
+import './ShoppingCartMaster.less'
+import { Button } from 'antd'
+import { Statistic, Row, Col } from 'antd'
+
+export const ShoppingCart = ({ currency = 'TL', price = 5, cart }) => {
```

Comment:
In my opinion, It would be better to get products in the cart as a prop here. Then we can make a request to the backend in the `ShoppingCartPage.js` and send it to this component as a prop. It's just a design choice though.

Reply:
Now that I think, If we are going to show some type loading icon, it might be better to do the request here and pull the products from inside this component.